cfme/scripting/setup_env.py

```diff
@@ -148,19 +153,30 @@ def setup_replication_env(cfme_version, provider, lease, sprout_poolid, desc):
     print("Configuring replicated environment")
     ip0 = apps[0].hostname
     ip1 = apps[1].hostname
+
     opt = '5' if cfme_version >= "5.8" else '8'
     command_set0 = ('ap', '', opt, '1', '1', 'y', '1', 'n', '99', pwd,
         TimedCommand(pwd, 360), '')
     apps[0].appliance_console.run_commands(command_set0)
     apps[0].wait_for_evm_service()
     apps[0].wait_for_web_ui()
     print("Global region appliance provisioned and configured {}".format(ip0))
+
     command_set1 = ('ap', '', opt, '2', ip0, '', pwd, '', '1', 'y', '1', 'n', '1', pwd,
         TimedCommand(pwd, 360), '')
     apps[1].appliance_console.run_commands(command_set1)
     apps[1].wait_for_evm_service()
     apps[1].wait_for_web_ui()
     print("Remote region appliance provisioned and configured {}".format(ip1))
+
+    if remote_worker:
+        ip2 = apps[2].hostname
```

Comment:
Please add print statement here that remote worker provisioning/configuration starting